scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -622,11 +668,12 @@ object Execution {
                   Future.successful(Map.empty)
               }
 
-            failFastZip(otherResult, localFlowDefCountersFuture).map {
-              case (lCounters, fdCounters) =>
-                val summedCounters = (fdCounters :: lCounters).reduce(_ ++ _)
-                (fn(conf, mode), summedCounters)
-            }
+            val bothFutures = failFastZip(otherResult, localFlowDefCountersFuture)
```

Comment:
Ya I was just a little confused by what was going on here in the old code. I think it makes sense now. I think the concept of "we" and "someoneElse" was not clear to me before.